.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Device (please complete the following information):**
```

Comment:
Anything else we might want to capture here to help?

Reply:
May be 
Is it device specific or happening on chrome ,  1 or more devices .
How consistent is the issue i.e., is it reproducible every time 
Have you seen this journey working before ?

Reply:
Great ideas @alphonsamuttaluri. I'll take a look at adding 👍